tsl/src/gapfill/exec.c

```diff
@@ -178,6 +189,182 @@ gapfill_state_create(CustomScan *cscan)
 	return (Node *) state;
 }
 
+/*
+ * lookup operator name in pg_operator
+ *
+ * returns empty string for unknown operator
```

Comment:
I think this should really return `NULL` or `elog(ERROR, ...` on an unknown operator. Though it does make the calling code a little more verbose in this case, we've already got too many calling conventions.